### PR TITLE
[Naitei] ft:View_All_User_Admin

### PR DESCRIPTION
**Pull request by ThaiKhacDuoc**
## Related Tickets
- ticket redmine

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

image

### CODE REVIEW

**Comment by ThaiKhacDuoc** [comment]
ready

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Admin::UsersController < AdminController
+  def index
+    @pagy, @users = pagy(User.all, items: Settings.page_size)
```
Comment (ghost) [inline app/controllers/admin/users_controller.rb]: viết scope cho user đi em

**Review by ghost — COMMENTED** [review]
cmt

**Comment by ThaiKhacDuoc** [comment]
ready

**Pull request merged**